app/controllers/api/v1/participant_declarations_controller.rb

```diff
@@ -10,11 +10,18 @@ class ParticipantDeclarationsController < Api::ApiController
       include ApiFilter
 
       def create
-        params = HashWithIndifferentAccess.new({ cpd_lead_provider: }).merge(permitted_params["attributes"] || {})
+        if permitted_params.dig("attributes", "declaration_type")
```

Comment:
used this to get path to run some `spec/request/api/v1/participants_spec.rb` to craft the errors message response serialisation.